### ISSUE TITLE
Useless encryption

### ISSUE THREAD

**Issue by agitka** [VKCOM/vk-ios-sdk#33]
In `VKAccessToken.m` you declare `static NSString *const TOKEN_KEY = @"VK is the best";`, but this is not the best and addition useless.
Because If someone uses cocoapod, it hasn't access to source and can't define own value without conflict when updating pods(or pods isn't under vcs). So that key is unavailable to modified without pain. It means every your user have 'VK is the best' as a key as a result encryption is useless. 
In example project you write `static NSString *const TOKEN_KEY = @"my_application_access_token";` in `VKStartScreen.h` 
but it's pointlessly and doesn't override value of TOKEN_KEY.
In conclusion, if you want to give a option for security, you must pull TOKEN_KEY from constants of the project. Now encryption is useless and only eats cpu ticks.
